docs/dashboard-spec.md

```diff
@@ -14,6 +14,31 @@ packs/<pack_name>/dashboards/apache.json
 
 The schema adheres to the `DashboardInput` used in the [dashboardCreate](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/dashboards-api-migration-insights-api-nerdgraph/#dashboard-create) mutation in NerdGraph.
 
+
+```yaml
+####################################################################################
+# Metadata
+####################################################################################
+
+# the name of the Dashboard
+name: string, required
+
+# A short description for this Dashboard
+description: string, required
+
+# List of screenshot URLs related to this Dashboard
+type: list, required
+
+# URL of Dashboard JSON definition
+url: string, required
```

Comment:
Same comment as the Alerts URL.

Reply:
it's defined as URL of Dashboard JSON definition

<img width="365" alt="Screen Shot 2021-08-19 at 9 12 03 AM" src="https://user-images.githubusercontent.com/1395158/130094090-bc64271c-b9d8-4136-850c-b03f18f3842a.png">

Reply:
Same thing here - the service adds this URL field automatically. We shouldn't be asking users to enter it.

Reply:
I'll remove it.